introduction.Rmd

```diff
@@ -0,0 +1,44 @@
+# Introduction
+
+R is a popular, versatile (and free!) programming language for data analysis.  This exercise will introduce you to some of the tools and ideas used for organizing data in R (such as variables, vectors, and data.frames). You will use an R tutorial package called `swirl`, to learn some common R commands for working with data. By the end of the exercise, you will use your R programming skills to answer some questions about the `us_contagious_diseases` dataset.
+
+This exercise will introduce you to two important data science tools:
+
+- **R programming language**: R is a programming language used by scientists to manipulate, visualize, and analyze data.  It is one of the most popular tools for data analysis.
+- **R Studio**:  RStudio helps you write and manage R projects.  Similar to how programs like Microsoft Word help you write documents, RStudio helps you carry out data analysis projects with R.
+
+## What is R?
```

Comment:
- Title doesn't quite match the text ... perhaps shorten to summarize 1.1.X and create a new H3 "What is a programming language"?
- How about ordering example languages: R, Python, C/C++/Rust, HTML/JavaScript
- Screams for a slidedeck ... for v2.0 :)

Reply:
> How about ordering example languages: R, Python, C/C++/Rust, HTML/JavaScript

What's the thought behind reordering?  I ordered them by how hard it was to explain what they're good for, haha, since the point I was trying to make is that different languages are good for different things.  I'm not sure how well that point is coming across?

But I put
- HTML/JavaScript first because "building websites" is short, straightforward, and relatable.
- C second because  "building software that needs to run really quickly" is a little more abstract
- Python last because I was having a really hard time coming up with a short explanation of what its "uniquely" good for.  I mean, I really like Python and I think it's biggest strength is the whole "Python is the second best language for everything" - it can do *everything*, usually with some pretty short, understandable code.  I think Python is not a great example for the point I was trying to make, because it's specialty is being easy to code in.  But it's one that a lot of people have heard of, especially around data analysis, so it felt wrong to not mention it.

I just added Swift/Kotlin to help make the "different languages for different purposes" point clearer, and free up Python from having to carry that point.  But I'm not sure what your thought was behind the reorder, so maybe that's still something to address?

Reply:
My suggested order is based on relevance to the audience (presumably future data analysts learning R).  Love the addition of Swift/Kotlin!